app/models/project.rb

```diff
@@ -25,4 +25,10 @@ def repo_path
 
     [url_segments[1], url_segments[2]].join("/")
   end
+
+  def get_clones
+    clones.joins(:user)
+        .where(project: self)
+        .order('users.nickname asc, users.email asc, clones.created_at desc')
+  end
```

Comment:
this allows us to sort the student clones by their name in ascending order, and then by the clone creation in descending order (newest first)